converter_schwab.py

```diff
@@ -0,0 +1,169 @@
+                    raise RuntimeError(f"Found duplicated RSU deposit event: {tmp}")
+                schwab_rsu_deposit_events[key] = tmp
+
+            elif e["Action"] == "Dividend" and e["Description"] == "Credit":
+                tmp = DividendRow.from_schwab_json(e)
+                if len(schwab_dividend_events) > 0 and isinstance(
+                    schwab_dividend_events[-1], TaxWithholdingRow
+                ):
+                    tax = schwab_dividend_events.pop(-1)
+                    tmp.tax_withholding = tax.amount
+                    schwab_dividend_events.append(tmp.to_dict())
+                else:
+                    schwab_dividend_events.append(tmp)
+
+            elif e["Action"] == "Sale" and e["Description"] == "Share Sale":
+                schwab_sell_events.append(SellOrderRow.from_schwab_json(e).to_dict())
+
+            elif (
+                e["Action"] == "Wire Transfer"
+                and e["Description"] == "Cash Disbursement"
+            ):
+                schwab_wire_events.append(
+                    CurrencyConversionRow.from_schwab_json(e).to_dict()
+                )
+
+            elif e["Action"] == "Tax Withholding" and e["Description"] == "Debit":
+                tmp = TaxWithholdingRow.from_schwab_json(e)
+                if len(schwab_dividend_events) > 0 and isinstance(
+                    schwab_dividend_events[-1], DividendRow
+                ):
+                    div = schwab_dividend_events.pop(-1)
+                    div.tax_withholding = tmp.amount
+                    schwab_dividend_events.append(div.to_dict())
+                else:
+                    schwab_dividend_events.append(tmp)
+
+            else:
+                act = e["Action"]
+                des = e["Description"]
+                sym = e["Symbol"]
+                date = e["Date"]
+                warnings.warn(f"skipping {act} on {date} ({sym}: {des})")
+
+    if len(schwab_rsu_lapse_events) != len(schwab_rsu_deposit_events):
+        raise RuntimeError(
+            f"Number of RSU Lapses {len(schwab_rsu_lapse_events)} does not match number of RSU deposits {len(schwab_rsu_deposit_events)}"
+        )
+    elif len(schwab_rsu_lapse_events) > 0:
+        for key, rsu in schwab_rsu_deposit_events.items():
+            if key in schwab_rsu_lapse_events:
+                rsu_lapse = schwab_rsu_lapse_events[key]
+            else:
+                raise ValueError(
+                    f"RSU Deposit {key} does not have a matching Lapse Event"
+                )
+            # schwab applies splits on historical lapse data but not on deposits
+            # thus, use this difference to determine split factor on-the-fly
+            # based on the split factor, we then can rely on the gross quantity
+            # in the lapse event while the prices for the deposit event are already
+            # correct
+            split_factor = rsu_lapse.net_quantity / rsu.net_quantity
```

Comment:
Class `Float64Dtype` does not define `__truediv__`, so the `/` operator cannot be used on its instances